conference/tickets.py

```diff
@@ -40,7 +38,14 @@ def count_number_of_sold_training_tickets_including_combined_tickets():
         frozen=False,
         orderitem__order___complete=True,
     ).filter(
-        Q(fare__code__regex=FARE_CODE_REGEXES["variants"][FARE_CODE_VARIANTS.TRAINING])
-        | Q(fare__code=FARE_CODE_REGEXES["variants"][FARE_CODE_VARIANTS.COMBINED])
```

Comment:
I've only noticed this kinda late, but the second condition doesn't use the `regex` comparison - we could probably still go with using regexes.